testing/src/test/java/io/stargate/it/http/RestApiv2Test.java

```diff
@@ -1358,7 +1358,7 @@ public void getColumnsPretty() throws IOException {
             value ->
                 assertThat(value)
                     .isEqualToComparingFieldByField(
-                        new ColumnDefinition("firstName", "Varchar", false)));
+                        new ColumnDefinition("firstName", "varchar", false)));
```

Comment:
I am not sure about one thing:
what type set is this? I suspect in C* it is `text` here it is `varchar`?
sorry if answer is obvious

Reply:
Correct. Table is created as `text` but we only have access to `varchar` on the way out.

Reply:
`varchar` and `text` are synonymous in the C* protocol.